ui/app/models/database/role.js

```diff
@@ -12,13 +12,13 @@ export default Model.extend({
     label: 'Role name',
   }),
   database: attr('array', {
-    label: '',
```

Comment:
I think the intent here may have been to only render a smaller label instead of a section header. But `FormField` generates a `labelString` if one isn't passed in, so two labels were rendering. 

Before 
<img width="500" alt="Screen Shot 2022-09-26 at 12 20 11 PM" src="https://user-images.githubusercontent.com/68122737/192361840-15eb215e-67a4-4a67-8c39-08f1da46587b.png">

Now 
<img width="500" alt="Screen Shot 2022-09-26 at 12 20 26 PM" src="https://user-images.githubusercontent.com/68122737/192361874-55c4b686-c66c-4901-807d-01db2f087a41.png">

Reply:
The "Now" looks correct, the only difference from the designs below is that we have Role name first in the page view and not in the designs 🤷‍♀️

![image](https://user-images.githubusercontent.com/6618863/192398137-bd78be3f-badd-4bfa-b465-f71a9b8eeedb.png)

Reply:
Thanks for the screenshot of designs! It also looks like they're not separate sections in the designs (but in the UI they're separated)

Reply:
It seems to me if we are creating a role then _Role name_ should be the first field.

Reply:
Yeah - I agree. Going to leave as is (maybe this was also decided during development and the designs weren't updated)